src/app/app.html

```diff
@@ -9,6 +9,10 @@
     <ion-content>
       <ion-list role="navigation" id="navigation-menu" aria-label="Peeveetrack Menu">
         <button role="link" menuClose ion-item *ngFor="let p of pages" (click)="openPage(p)">
+        <ion-icon name="ios-heart-outline" *ngIf="p.title == 'Favorites'"></ion-icon>
```

Comment:
@dfaulken makes a great point.

@mboneil10, I think you're very close to the optimal solution, but here's what I'd do: edit the `pages` array in AppComponent to include the icon name for each entry.

For example:

```javascript
this.pages = [
  {
    title: 'Favorites',
    component: MyBusesComponent,
    icon: 'ios-heart-outline'
  }
...
]
```
This way you only need one icon tag inside the `*ngFor`:

```html
<ion-icon name="p.icon"></ion-icon>
```
And that should do it!